dbt/profiles/profiles.yml

```
@@ -19,3 +19,14 @@ default:
       threads: 6
+elementary:
+  outputs:
+    default:
+      type: postgres
+      host: "{{ env_var('DBT_HOST', 'localhost') }}"
+      user: "{{ env_var('DBT_USER', 'postgres') }}"
+      password: "{{ env_var('DBT_PASSWORD', 'password') }}"
+      port: "{{ env_var('DBT_PORT', 5432) | int }}"
+      dbname: "{{ env_var('DBT_DBNAME', 'covoiturage_production') }}" # or database instead of dbname
+      schema: "{{ env_var('DBT_SCHEMA', '') }}"
+      threads: 6
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)